R/rd-family.R

```diff
@@ -1,3 +1,15 @@
+topics_process_family_prefix <- function(family) {
+
+  # read custom family prefix from roxygen meta
+  opts <- roxy_meta_read("family.prefix")
+  if (!is.null(opts[[family]]))
```

Comment:
Where should we check that this is a list?

Reply:
I think it's okay for `rd_family_title` to just be a named 'something' -- doesn't necessarily have to be a list. (We just enforce that the metadata object itself is a list). We could be stricter if you prefer though.

Reply:
I'm slightly worried that if it's not a list (e.g. it's a function) you'll get an error message that will be extremely difficult to diagnose. I'm not sure if it's worth handling that rare case.